ext/configuration.c

```diff
@@ -181,11 +181,11 @@ bool ddtrace_config_minit(int module_number) {
     }
 
 #ifndef _WIN32
-    // Sidecar is currently broken - no traces sent. Investigation pending, background sender just works though.
+    // Background sender does not send a Content-Length header, but sidecar does. Force-enable it thus, as the background sender does not work at all.
     if (getenv("AWS_LAMBDA_FUNCTION_NAME")) {
```

Comment:
There is a similar condition in `appsec/src/extension/configuration.c`, shouldn't it be updated also?

Reply:
Oh, there is? Well, then yes, it should :-D

Reply:
Done :-)